### ISSUE TITLE
Open Watcom not compiling with WinXP

### ISSUE THREAD

**Issue by MikeyG** [open-watcom/open-watcom-v2#457]
I ran my compile setup for the first time in a while and received the following error during build:

==================== 17:41:02 c:\open-watcom-v2\bld\whpcvt ====================
Copying 'c:\open-watcom-v2\bld\whpcvt\binbuild\whpcvt.exe' to 'c:\open-watcom-v2
\build\binbuild\whpcvt.exe'...
==================== 17:41:02 c:\open-watcom-v2\bld\whpcvt ====================
**** Building the wbind bootstrap
================= 17:41:02 c:\open-watcom-v2\bld\win386\wbind =================
============ 17:41:03 c:\open-watcom-v2\bld\win386\wbind\binbuild =============
cc wbind.obj
link wbind.exe
================= 17:41:03 c:\open-watcom-v2\bld\win386\wbind =================
Copying 'c:\open-watcom-v2\bld\win386\wbind\binbuild\wbind.exe' to 'c:\open-watc
om-v2\build\binbuild\bwbind.exe'...
================= 17:41:03 c:\open-watcom-v2\bld\win386\wbind =================
**** Building the dlgprs bootstrap
================ 17:41:03 c:\open-watcom-v2\bld\browser\dlgprs ================
=========== 17:41:03 c:\open-watcom-v2\bld\browser\dlgprs\binbuild ============
link dlgprs.exe
Error! E2008: cannot open bind.obj : No such file or directory
Error! E2008: cannot open dialog.obj : No such file or directory
Error! E2008: cannot open main.obj : No such file or directory
Error! E2008: cannot open scanner.obj : No such file or directory
Error! E2008: cannot open styles.obj : No such file or directory
Error! E2008: cannot open bndytab.obj : No such file or directory
Error! E2008: cannot open dlgytab.obj : No such file or directory
Error! E2008: cannot open chfile.obj : No such file or directory
Error! E2008: cannot open chbffile.obj : No such file or directory
Error! E2008: cannot open mempool.obj : No such file or directory
================ 17:41:03 c:\open-watcom-v2\bld\browser\dlgprs ================
Can not open 'c:\open-watcom-v2\bld\browser\dlgprs\binbuild\dlgprs.exe' for read
ing: No such file or directory
Build failed

**Comment by jmalak** [comment]
What bootstrap compiler you use?

**Comment by jmalak** [comment]
it looks like a problem with C++ compiler.

**Comment by MikeyG** [comment]
I checked, I am using:

Open Watcom C x86 32-bit Optimizing Compiler
Version 2.0 beta Apr 27 2014 22:30:10 (32-bit)
Copyright (c) 2002-2014 The Open Watcom Contributors. All Rights Reserved.

Should I stick with using 1.9?

**Comment by jmalak** [comment]
Yes, OW 1.9 compiler is supported as bootstrap compiler.

Now are you using compiler from Travis CI build ?

**Comment by jmalak** [comment]
Try to download latest OW build from Github repository.
May be the version which you use is somehow damaged.

**Comment by MikeyG** [comment]
I downloaded the old OW 1.9 version and compiled ....
```
awk usagej.gh
rc wresui.res
rc exever.res
link wdis.exe
bind res wdis.exe
bind wresui wdis.exe
=================== 20:51:44 c:\open-watcom-v2\bld\ndisasm ====================
Copying 'c:\open-watcom-v2\bld\ndisasm\binbuild\wdis.exe' to 'c:\open-watcom-v2\
build\binbuild\bwdis.exe'...
=================== 20:51:44 c:\open-watcom-v2\bld\ndisasm ====================
**** Building the dmpobj bootstrap
==================== 20:51:44 c:\open-watcom-v2\bld\dmpobj ====================
=============== 20:51:44 c:\open-watcom-v2\bld\dmpobj\binbuild ================
cc dmpobj.obj
..\c\dmpobj.c(185): Warning! W107: Missing return value for function 'main'
Error(E42): Last command making (dmpobj.obj) returned a bad status
Error(E02): Make execution terminated
<wmake -h -f ../binmake bootstrap=1> => non-zero return: 2
Build failed
C:\open-watcom-v2>
```

I'll keep trying to figure out if this is just my build setup causing problems.

**Comment by jmalak** [comment]
Sorry, it is another problem (with OW 1.9 compiler).
It is fixed by 1f0b2b528d256a4d7bbd28f6393e6676dfca800a commit.

**Comment by MikeyG** [comment]
Jiri - That seemed to fix everything, I am closing this one - thanks.
